.yarnrc.yml

```diff
@@ -1,3 +1,3 @@
 nodeLinker: node-modules
 
-yarnPath: .yarn/releases/yarn-3.1.1.cjs
\ No newline at end of file
+yarnPath: .yarn/releases/yarn-3.6.0.cjs
```

Comment:
I'd keep the changes scoped to the tutorial rather than including updating related yarn files. This includes the deletion of `.yarn/releases/yarn-3.1.1.cjs`. You can open a subsequent PR for those changes please.

Reply:
Ah, it probably automatically got updated when I ran the commands. I've reverted it back to the previous version 👍🏻